Navigation.css

```diff
@@ -0,0 +1,98 @@
+:root{
+    --primary-color:turquoise;
+    --secondary-color:rgb(155, 173, 238);
+    --tertiary-color: skyblue;
+
+}
+
+body{
+    position: relative;
+    margin-left: 4px;
+    width: 100%;
+}
+
+.body{
+    width: 100%;
+}
+.container{
+    padding: 3px;
+    display: flex;
+    gap: 5px;
```

Comment:
i will suggest not to use px you can use rem or em

Reply:
but we cannot use rem when unit is below 10 px